src/array/without.js

```diff
@@ -0,0 +1,11 @@
+import { pull } from './pull'
+
+/**
+ * This method is an alias of {@link array.pull}.
```

Comment:
JsDoc doesn't handle aliases...
On lodash documentation jsdoc has been tweeked to manage aliases, there is [an issue at jsdoc] (by lodash main commiter) but it hasn't been taken care of...
As it is it don't think it will work with aliases.